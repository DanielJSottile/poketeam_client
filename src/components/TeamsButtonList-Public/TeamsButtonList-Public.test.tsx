import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
//import { render } from '@testing-library/react';
import TeamsButtonListPublic from './TeamsButtonList-Public';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<TeamsButtonListPublic/>', (): void => {
  it('renders without crashing', (): void => {
    const div: HTMLDivElement = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <TeamsButtonListPublic />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it('snapshot', (): void => {
    const wrapper = shallow(<TeamsButtonListPublic />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
