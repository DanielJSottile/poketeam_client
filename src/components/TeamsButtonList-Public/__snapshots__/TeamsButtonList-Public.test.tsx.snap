// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamsButtonListPublic/> snapshot 1`] = `
<Fragment>
  <section
    className="folders-list"
  >
    <div>
      <div
        className="pagebutton"
      >
        <button
          onClick={[Function]}
        >
          <i
            className="fas fa-arrow-circle-right"
          />
           
          Go to Next 10 Teams
        </button>
      </div>
      <span>
        Current Teams: 1 - 10
      </span>
      <h3>
        Teams:
      </h3>
    </div>
    <div>
      <div
        className="pokeball-div"
      >
        <PokeballLoader />
        <LoadingWhite />
        <h3
          className="hint"
        >
          (Hint: There May Be No Teams)
        </h3>
      </div>
    </div>
    <div>
      <span>
        Current Team: 
      </span>
    </div>
  </section>
</Fragment>
`;
